### PR TITLE
Enable dashboard element ganache testing resolve

### PR DESCRIPTION
**Pull request by acarnagey**

### CODE REVIEW

File: packages/dashboard/next.config.js [hunk 1]
```diff
@@ -6,14 +6,16 @@ module.exports = {
...
   experimental: { esmExternals: 'loose', outputFileTracing: true },
+  webpack5: false,
   // see https://github.com/vercel/vercel/issues/2569#issuecomment-514865342
   webpack(config, { isServer }) {
     if (isServer) {
       config.resolve.mainFields = ['module', 'main'];
 
       // Fix all packages that this change breaks:
-      // config.resolve.alias['node-fetch'] = 'node-fetch/lib/index.js';
+      config.resolve.alias['node-fetch'] = 'node-fetch/lib/index.js';
```
Comment (OR13) [inline packages/dashboard/next.config.js:18]: webpack :(

File: packages/dashboard/pages/api/did/[did].ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
+import getElement from '../../../server/getElement';
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<any>
+) {
+  const { did } = req.query;
+  const element = await getElement();
```
Comment (OR13) [inline packages/dashboard/pages/api/did/[did].ts:9]: I think there is a decorator for this... 

https://github.com/transmute-industries/sidetree.js/blob/main/packages/dashboard/pages/api/1.0/identifiers/%5Bdid%5D/index.tsx#L27

https://github.com/transmute-industries/sidetree.js/blob/main/packages/dashboard/middleware/sidetree.ts

File: packages/dashboard/pages/resolve.tsx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+          <Grid container spacing={2}>
+            <Box sx={{ width: 1 }} m={2}>
+              <form onSubmit={resolveDid}>
+                <FormControl fullWidth>
+                  <InputLabel htmlFor="did">DID</InputLabel>
+                  <Input
+                    id="did"
+                    placeholder="did:elem:ropsten:123"
+                    value={did}
+                    onChange={(e) => setDid(e.target.value)}
```
Comment (OR13) [inline packages/dashboard/pages/resolve.tsx:53]: probably we can just redirect here: https://github.com/transmute-industries/sidetree.js/blob/main/packages/dashboard/pages/%5Bdid%5D/index.tsx

File: packages/dashboard/server/getElement.ts [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  return cas;
+};
+
+async function createElement() {
+  const ledger: any = await getTestLedger();
+  const cas: any = await getTestCas();
+  const element = new Element(config as any, config.versions, ledger, cas);
+  await element.initialize(
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    { info: () => {}, warn: () => {}, error: () => {} },
```
Comment (OR13) [inline packages/dashboard/server/getElement.ts:27]: XD

**Comment by OR13** [comment]
Screen Shot 2021-11-30 at 10 17 10 AM

CI is failing for the same reason it was before, web3 module is somehow thinking it is running in electron, when building the app.

File: packages/dashboard/server/getElement.ts (hunk 4 above)
Comment (acarnagey) [reply packages/dashboard/server/getElement.ts:27]: Still didn't work :/ I'm getting a ton of noise in the logs

**Review by OR13 — DISMISSED** [review]
Looks good, let tackle comments in future PRs.

**Review by OR13 — APPROVED** [review]

**Pull request merged**